src/components/PayPalButtons.js

```diff
@@ -141,6 +141,10 @@ PayPalButtons.propTypes = {
      * Called when the buyer changes their shipping address on PayPal.
      */
     onShippingChange: PropTypes.func,
+    /**
+     * Called when the button need to be re-rendered.
+     */
+    forceReRender: PropTypes.string,
```

Comment:
I don't think we need to limit this to a string. I think a number or boolean or any other value would be fine too.
```suggestion
    forceReRender: PropTypes.any,
```